src/components/DatePicker/style.ts

```diff
@@ -0,0 +1,44 @@
+import { css } from '@emotion/css';
+
+// size
+const smallSize = '0.25rem 0.5rem';
+const mediumSize = '0.5rem 1rem';
+const largeSize = '1.5rem 2rem';
+
+// status
+const successState = '2px solid #2D9D4D';
+const errorState = '2px solid #CD1212';
```

Comment:
we need to have different standard on something different.. like this one.. this variables will not changed forever right? if that is the case.. people tends to use `SCREAMING_SNAKE_CASE` naming format.. (but you guys choose what the most suitable for it)
```suggestion
const SMALL_SIZE = '0.25rem 0.5rem';
const MEDIUM_SIZE = '0.5rem 1rem';
const LARGE_SIZE = '1.5rem 2rem';

// status
const SUCCESS_STATE = '2px solid #2D9D4D';
const ERROR_STATE = '2px solid #CD1212';
```
in this case when you saw it being used in you code.. you know that this is a fixed value that you define somewhere in the code and not processed by any functions or API or so on.